Dictionaries/srp_OCRFixReplaceList.xml

```diff
@@ -61,15 +61,14 @@
     <!-- Two or more consecutive '"' to one '"' -->
     <RegEx find="&quot;{2,}" replaceWith="&quot;" />
     <!-- Fix zero and capital 'o' ripping mistakes -->
-    <RegEx find="([0-9])O" replaceWith="${1}0" />
-    <RegEx find="([0-9])\.O" replaceWith="$1.0" />
-    <RegEx find="\b0([A-Za-z])" replaceWith="O$1" />
+    <RegEx find="(?&lt;=[0-9]\.?)O" replaceWith="0" />
+    <RegEx find="\b0(?=[A-Za-z])" replaceWith="O" />
```

Comment:
Since Serbian isn't based in `ASCII` I think it would be better idea to include: `zčđšž` as well. What do you think?